eng/scripts/Language-Settings.ps1

```diff
@@ -6,7 +6,35 @@ $packagePattern = "Cargo.toml"
 $GithubUri = "https://github.com/Azure/azure-sdk-for-rust"
 $PackageRepositoryUri = "https://crates.io/crates"
 
-function Get-AllPackageInfoFromRepo ([string] $ServiceDirectory) {
+function SetPackageVersion ($PackageName, $Version, $ServiceDirectory, $ReleaseDate, $ReplaceLatestEntryTitle=$true)
+{
+  if($null -eq $ReleaseDate)
+  {
+    $ReleaseDate = Get-Date -Format "yyyy-MM-dd"
+  }
+  & "$EngDir/scripts/Update-PackageVersion.ps1" -ServiceDirectory $ServiceDirectory -PackageName $PackageName `
+      -NewVersionString $Version -ReleaseDate $ReleaseDate -ReplaceLatestEntryTitle $ReplaceLatestEntryTitle
+}
+
+function GetExistingPackageVersions ($PackageName, $GroupId=$null)
+{
+  try {
+    $PackageName = $PackageName.ToLower()
```

Comment:
This probably doesn't hurt, but shouldn't these be all lowercase anyway? Wondering if maybe we should let it error otherwise. Or is this purely based on user input and we want to normalize it?

Reply:
It would be based on user input.  The user will call `Prepare-Release.ps1 --ServiceDirectory foo --PackageName azure_foo`.  I normalize it here.